vars/common.yml

```diff
@@ -2,11 +2,26 @@
 aws_region: eu-west-1
 
 api:
-  rds_port: 5432
-  db_user: "digitalmarketplace"
-  db_name: "digitalmarketplace_api"
-  db_instance_type: db.m1.small
-  db_allocated_storage: 5
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
+
+search_api:
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
+
+admin_frontend:
+  instance_type: t2.micro
+  min_instance_count: 1
+  max_instance_count: 2
+
+database:
+  port: 5432
+  user: "digitalmarketplace"
+  name: "digitalmarketplace_api"
+  instance_type: db.t2.micro
+  allocated_storage: 5
 
 elasticsearch:
   port: 9200
```

Comment:
It already is but it just has `instance_count` instead of `min_` and `max_` because we can't autoscale it